### PR TITLE
Adding Perl library "Log::Log4perl"

### PR DESCRIPTION
**Pull request by tdegeus**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

This recipe is based on https://github.com/conda-forge/perl-xml-parser-feedstock

### CODE REVIEW

**Comment by tdegeus** [comment]
How should I handle the fact the the package exists on bioconda @conda-forge/bioconda-recipes ?

**Comment by tdegeus** [comment]
Read for review @conda-forge/help-perl

File: recipes/perl-log-log4perl/LICENSE [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Copyright 2002-2013 by Mike Schilli <m@perlmeister.com> and Kevin Goess <cpan@goess.org>.
```
Comment (marcelotrevisani) [inline recipes/perl-log-log4perl/LICENSE:1]: You can delete this file, the license is already being packaged with the source files

Comment (tdegeus) [reply recipes/perl-log-log4perl/LICENSE]: Makes sense!

File: recipes/perl-log-log4perl/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{% set version = "1.49" %}
+
+package:
+  name: perl-xml-parser
```
Comment (tdegeus) [inline recipes/perl-log-log4perl/meta.yaml:4]: This should be changed into `  name: perl-log-log4perl`

**Pull request merged**